### DISCUSSION TITLE
Won't work for organizations

### DISCUSSION THREAD

**Discussion by ThirtySomething** [lowlighter/metrics#1319] [Q&A] [answered]
Hi there,

I've been successful in setting up this for my [own account](https://github.com/ThirtySomething). As I've tried to do this for an [organization account](https://github.com/SourceMonitor) in the same way, I've got errors. Might it be possible that this will not work for organizations?

Regards

ThirtySomething

**Comment by ThirtySomething** [comment, marked as answer]
Uh, I'm sorry, this works for organizations, too. There is another '[kind of widget](https://github.com/anuraghazra/github-readme-stats)' of another GitHub user which won't work. You may close this discuttion.

**Reply by lowlighter** [reply]
Note that not all plugins support organization though

You can check whether a plugin supports organization on their respective documentation within their header table:
![image](https://user-images.githubusercontent.com/22963968/207770309-e1e52e74-7dc6-423a-b555-c2f073990214.png)
